ui/src/app/pages/login/sign-in.tsx

```diff
@@ -272,6 +273,13 @@ export class SignInImpl extends React.Component<SignInProps, SignInState> {
     if (index === steps.length) {
       throw new Error('No sign-in steps remaining after step ' + currentStep);
     }
+    if (
+      steps[index] === SignInStep.ACCOUNT_DETAILS &&
+      new Date() >= INTL_USER_SIGN_IN_CHECK &&
+      profile.address.country !== Country.US
```

Comment:
Add a helper method, or comments, that link this to the other places this particular logic is implemented.